README.md

```diff
@@ -19,6 +19,17 @@ Please refer to [install.md](docs/install.md) for installation and dataset prepa
 
 Please see [getting_started.md](docs/getting_started.md) for the basic usage. There are also tutorials for [finetuning models](docs/tutorials/finetune.md), [adding new dataset](docs/tutorials/new_dataset.md), [designing data pipeline](docs/tutorials/data_pipeline.md), and [adding new modules](docs/tutorials/new_modules.md).
 
+## Models compression
+
+This project allows to make compression of the models by [NNCF](https://github.com/openvinotoolkit/nncf_pytorch) framework.
```

Comment:
allows you to make or allows making

Reply:
Sorry, Ilya, but I found sufficiently many examples with "allows to make" without "you":  
https://www.google.com/search?q=%22allows+to+make%22  
https://context.reverso.net/translation/english-russian/%22allows+to+make%22  

I'd better keep the current form of the sentence...

Reply:
https://www.grammaring.com/to-infinitive-or-gerund-advise-recommend-allow-permit-forbid-require

Reply:
Yes, I've found it already too.

Fixed in one of my last commits.